client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -24,8 +25,7 @@ import {
 class DisconnectJetpackButton extends Component {
 	state = { dialogVisible: false };
 
-	handleClick = event => {
-		event.preventDefault();
```

Comment:
👍 

Curious why this was here, I can't find any differences between the [branch](https://calypso.live/sites?branch=update/conditionally-enable-jetpack-disconnect-flow&flags=-manage/site-settings/disconnect-flow) with the flag disabled and staging.

Reply:
Looks like matter of habit to me. Or maybe there was a `href="#"` cursor workaround at some point, too.